book/src/validator-manager-move.md

```diff
@@ -0,0 +1,181 @@
+
+There will be two VCs in this example:
+
+- The *source* VC which contains the validators/keystores to be moved.
+- The *destination* VC which is to take the validators/keystores from the source.
+
+There will be two hosts in this example:
+
+- Host 1 (*"source host"*): Is running the `src-vc`.
+- Host 2 (*"destination host"*): Is running the `dest-vc`.
+
+The example will assume that all commands are run on Host 1. It also assumes
+that Host 1 is able to SSH to Host 2.
+
+In reality, many host configurations are possible. For example:
+
+- Both VCs on the same host.
+- Both VCs on different hosts and the `validator-manager` being used on a third host.
+
+### 1. Configure the Source VC
+
+The source VC needs to have the following flags at a mininum:
+
+- `--http`
+- `--unencrypted-http-transport`
+- `--http-address 127.0.0.1`
+- `--http-port 5062`
+- `--http-allow-keystore-export`
+
+Therefore, the source VC command might look like:
+
+```bash
+lighthouse \
+    vc \
+    --http \
+    --unencrypted-http-transport \
+    --http-address 127.0.0.1 \
+    --http-port 5062 \
+    --http-allow-keystore-export
+```
+
+### 2. Configure the Destination VC
+
+The destination VC needs to have the following flags at a mininum:
+
+- `--http`
+- `--unencrypted-http-transport`
+- `--http-address 127.0.0.1`
+- `--http-port 5062`
+- `--enable-doppelganger-protection`
+
+Therefore, the destination VC command might look like:
+
+```bash
+lighthouse \
+    vc \
+    --http \
+    --unencrypted-http-transport \
+    --http-address 127.0.0.1 \
+    --http-port 5062 \
+    --enable-doppelganger-protection
+```
+
+> The `--enable-doppelganger-protection` flag is not *strictly* required, however
+> it is recommended for an additional layer of safety. It will result in 3-4
+> epochs of downtime for the validator after it is moved, which is generally an
+> inconsequential cost in lost rewards or penalties.
+> 
+> Optionally, users can add the `--http-store-passwords-in-secrets-dir` flag if they'd like to have
+> the import validator keystore passwords stored in separate files rather than in the
+> `valdiator-definitions.yml` file. If you don't know what this means, you can safely omit the flag.
+
+### 3. Obtain the Source API Token
+
+The VC API is protected by an *API token*. This is stored in a file on each of the hosts. Since
+we'll be running our command on the destination host, it will need to have the API token for the
+source host on its file-system.
+
+On the **source host**, find the location of the `api-token.txt` file and copy the contents. The
+location of the file varies, but it is located in the "validator directory" of your data directory,
+alongside validator keystores. For example: `~/.lighthouse/mainnet/validators/api-token.txt`.
+
+Copy the contents of that file into a new file on the **destination host** at `~/src-token.txt`. The
+API token should be similar to `api-token-0x03eace4c98e8f77477bb99efb74f9af10d800bd3318f92c33b719a4644254d4123`.
+
```

Comment:
Added the configurations required if both VCs are in the same host.
```suggestion
If both VCs are on the same host, you can skip copying the `api-token.txt` to the destination host, since now both `api-token.txt` files reside on the same host. You only have to identify the directory of the `api-token.txt` of both VCs. If you are unsure of the `api-token.txt` path, you can run (assuming the `http-port` used are 5062 and 5063):
`curl http://localhost:5062/lighthouse/auth` 
`curl http://localhost:5063/lighthouse/auth` 
```